src/main/main.ts

```diff
@@ -21,6 +21,9 @@ import { setupErrorRoutes } from './setup/routes/setupErrorRoutes';
 import { setupFeatureRoutes } from './setup/routes/setupFeatureRoutes';
 import { setupHomeRoutes } from './setup/routes/setupHomeRoutes';
 
+// @ts-ignore -- no types file
```

Comment:
Instead of adding a magic comment you can declare the types for the package. I've made a Pr showing how https://github.com/Crown-Commercial-Service/ccs-scale-cat-buyer-ui/pull/4629